README-dev.md

```diff
@@ -90,21 +90,16 @@ If you work on o1js, create a feature branch off of one of these base branches.
 
 **Default to `main` as the base branch**.
 
-The other base branches (`berkeley` and `develop`) are used only in specific scenarios where you want to adapt o1js to changes in the sibling repos on those other branches. Even then, consider whether it is feasible to land your changes to `main` and merge to `berkeley` and `develop` afterwards. Only changes in `main` will ever be released, so anything in the other branches has to be backported and reconciled with `main` eventually.
+The other base branches (`develop`) are used only in specific scenarios where you want to adapt o1js to changes in the sibling repos on those other branches. Even then, consider whether it is feasible to land your changes to `main` and merge to `develop` afterwards. Only changes in `main` will ever be released, so anything in the other branches has to be backported and reconciled with `main` eventually.
```

Comment:
thanks for updating this!

could you also fix the section that comes before this one, "Branch compatibility"? It seems it was copied out of place at one point in time: It talks about "one of these base branches" even though the different branches haven't been mentioned. Maybe you can combine that section with this one so that it makes more sense.